src/main/java/com/parasoft/demoapp/service/GlobalPreferencesDefaultSettingsService.java

```diff
@@ -1,25 +1,26 @@
 package com.parasoft.demoapp.service;
 
-import static com.parasoft.demoapp.config.ParasoftJDBCProxyConfig.*;
-
-import java.util.*;
-
+import com.parasoft.demoapp.config.WebConfig;
 import com.parasoft.demoapp.config.activemq.ActiveMQConfig;
+import com.parasoft.demoapp.config.datasource.IndustryRoutingDataSource;
 import com.parasoft.demoapp.model.global.preferences.*;
 import com.parasoft.demoapp.model.industry.RegionType;
+import com.parasoft.demoapp.util.BugsTypeSortOfDemoBugs;
+import com.parasoft.demoapp.util.RouteIdSortOfRestEndpoint;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
-import com.parasoft.demoapp.config.WebConfig;
-import com.parasoft.demoapp.config.datasource.IndustryRoutingDataSource;
-import com.parasoft.demoapp.util.BugsTypeSortOfDemoBugs;
-import com.parasoft.demoapp.util.RouteIdSortOfRestEndpoint;
+import java.util.*;
+
+import static com.parasoft.demoapp.config.ParasoftJDBCProxyConfig.*;
 
 @Service
 public class GlobalPreferencesDefaultSettingsService {
 
     public static final String HOST_WITHOUT_PORT = "http://localhost:";
 
+    public static final String GRAPHQL_ENDPOINT_URL = "http://localhost:8080/graphql";
```

Comment:
This url should not be fixed since the port can be customized. I suggest to change the name `GRAPHQL_ENDPOINT_URL` to `GRAPHQL_ENDPOINT_PATH` and  splice the url by `HOST_WITHOUT_PORT + webConfig.getServerPort() + GRAPHQL_ENDPOINT_PATH`.